[chocobo] Add GM commands for different coloured chocobos

zach2good opened the pull request:
<!-- remove space and place 'x' mark between square [] brackets or click the checkbox after saving to affirm: -->
**_I affirm:_**
- [x] I have paid attention to this example and will edit again if need be to not break the formatting, or I will be ignored
- [x] that I agree to LandSandBoat's [Limited Contributor License Agreement](https://github.com/LandSandBoat/server/blob/base/.github/CONTRIBUTOR_AGREEMENT.md), as written on this date
- [x] that I have **read the [Contributing Guide](https://github.com/LandSandBoat/server/blob/base/CONTRIBUTING.md)**
- [x] that I've _**tested my code and things my code changed**_ since the last commit in the PR, and will test after any later commits

Found these changes by Ivaar in an old DSP branch and ported them over to LSB for future use with my chocobo raising stuff.
https://github.com/DarkstarProject/darkstar/compare/master...Ivaar:chocobo

Mapitoto was already 99% prepared for this change.

Tested on stream, all works fine, don't ask about the magic numbers 🎉

Ivaar committed:
Add field chocobo information, db entry, and bindings

zach2good committed:
[chocobo] Clean up chocobo script and db migration

zach2good committed:
[chocobo] Hook up chocobo whistle to correctly show registered chocobo

zach2good committed:
[chocobo] Prepare Mapitoto for future Chocobo Whistle trading

TeoTwawki commented on scripts/globals/effects/mounted.lua:
```diff
@@ -6,11 +6,9 @@ require("scripts/globals/status")
+    -- Retail sends a music change packet (packet ID 0x5F) in both cases.
 
-    if effect:getPower() == 0 then
+    if effect:getPower() < 2 then
```
I think I missed this part on stream, value 1 is listed as the raptor on the quest so whats the deal here?

zach2good replied:
True, this is copy/paste from Ivaar's branch, I can test to see if this matters or not

zach2good replied:
Yeah this seems a bit off, I think there is a little more capturing to do. We are off-by-one _somewhere_ down the line for chocobo and mount stuff. Everything works currently, but it will want cleanup in the future. I'll do it as part of chocobo raising 👍

zach2good committed:
[chocobo] Clean up and comments

zach2good committed:
[ci] Remove lua lanes for Lua Style Checks

The unauthenticated git protocol on port 9418 is no longer supported.

https://github.blog/2021-09-01-improving-git-protocol-security-github/

zach2good commented:
This now contains a change we need for CI to work, so I'm yeeting this straight in
```
The unauthenticated git protocol on port 9418 is no longer supported.

https://github.blog/2021-09-01-improving-git-protocol-security-github/
```
